### ISSUE TITLE
feat(btp): Extend deployment script functionality to support different chain configs

### ISSUE THREAD

**Issue by andrii-kl** [icon-project/icon-bridge#641]
## Overview

Extend deployment script functionality to support different chain configs

New structure that I suggest to use:

devnet/docker/icon-bsc/scripts ->

/deploysc.sh  
...
/config (new folder) ->
      icon-lisb-bsc-test.sh
      icon-bsc-main.sh
      icon-lisb-snow-arctic.sh
      icon-snow-main.sh
      ......
      ......

## Story

As a [developer], I want [ to run deployment script with passing path to my chains config file ] so that [ deploy everything without any manual changes in the config file ]

**Comment by CyrusVorwald-ICON** [comment]
I agree about moving this to a new folder, but would much prefer that we transition away from bash scripts. Take a look at how Wormhole uses [Tilt](https://github.com/wormhole-foundation/wormhole/blob/dev.v2/Tiltfile) and [Kubernetes](https://github.com/wormhole-foundation/wormhole/tree/dev.v2/devnet). I think we should do pretty much the same thing.
